rails.md

```diff
@@ -37,52 +39,54 @@ Answer the following questions. First, without external resources. Challenge you
 3a. Consider the Rails routes below. Describe the responsibility of  each route.
 
 
-/users/       method="GET"     # :controller => 'users', :action => 'index'
+/users/       method="GET"     # :controller => 'users', :action => 'index' -  Display a list of all messages
 
-/users/1      method="GET"     # :controller => 'users', :action => 'show'
+/users/1      method="GET"     # :controller => 'users', :action => 'show' - Display a speciic message
 
-/users/new    method="GET"     # :controller => 'users', :action => 'new'
+/users/new    method="GET"     # :controller => 'users', :action => 'new' - Return an HTML form for creating a new messages
 
-/users/       method="POST"    # :controller => 'users', :action => 'create'
+/users/       method="POST"    # :controller => 'users', :action => 'create' - create a new message
 
-/users/1/edit method="GET"     # :controller => 'users', :action => 'edit'
+/users/1/edit method="GET"     # :controller => 'users', :action => 'edit' - return an HTML form editing a message
 
-/users/1      method="PUT"     # :controller => 'users', :action => 'update'
+/users/1      method="PUT"     # :controller => 'users', :action => 'update' - update a specific message
 
-/users/1      method="DELETE"  # :controller => 'users', :action => 'destroy'
+/users/1      method="DELETE"  # :controller => 'users', :action => 'destroy' - delete a specific message
 
 
 
 3b. Which of the above routes must always be passed params and why?
 
+The create action must alway be passeed params method to safely collect data from the form and update the database.
 
```

Comment:
The show, update and delete routes require the param of an id since the action must happen on a particular item.